app/templates/partials/summary/checkbox.html

```diff
@@ -1,5 +1,21 @@
+{% set not_answered = 'No answer provided' %}
 <ul class="u-m-no">
   {% for selected_answer in answer.value %}
-    <li>{{selected_answer}}</li>
+    <li>{{selected_answer[0]}}
+    {%- if answer.child_answer_value is none -%}
+        {%- else -%}
+          {% if selected_answer[1] %}
+            {%- if answer.child_answer_value == ''  -%}
+              <ul>
+                <li>{{not_answered}}</li>
```

Comment:
You could move this `%- if answer.child_answer_value == ''  -%}` to inside the `<li>`?
That would mean that the <ul><li> is not duplicated in the if/else